### PR TITLE
fix(viewtoggle): error that happens when the active option is removed

### PR DESCRIPTION
**Pull request by danethurber**
### What You're Solving

- https://pluralsight.slack.com/archives/C70HPSJPP/p1565630361034500

### CODE REVIEW

File: packages/viewtoggle/src/react/index.js [hunk 1]
```diff
@@ -40,6 +40,7 @@ const ViewToggle = React.forwardRef(({ onSelect, ...props }, forwardedRef) => {
 
   function renderActivePill() {
     const activeEl = React.Children.toArray(props.children)[activeIndex]
+    if (!activeEl) return null
```
Comment (jaketrent) [inline packages/viewtoggle/src/react/index.js]: Why doesn't the removal of an option (change in `props.children`) retrigger line 33 then 31 above?

If it did, `findActiveIndex` will default to index 0, where the active pill will sit.  

But in this solution, we simply don't get to see a pill.  Would that ui state even make sense?

Comment (danethurber) [reply packages/viewtoggle/src/react/index.js]: it does trigger an update but the rendering happens before the useEffect.

Comment (jaketrent) [reply packages/viewtoggle/src/react/index.js]: Just a followup to the 2nd question here: 

> But in this solution, we simply don't get to see a pill. Would that ui state even make sense?

First of all, is my observation accurate?  If so, that doesn't seem right to me.  Pulling code to try it...

Comment (jaketrent) [reply packages/viewtoggle/src/react/index.js]: Looks like when active el is removed, index 0 is reset, active pill bg still shown.  👍

**Pull request merged**